index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Desafio dos diamantes</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <select id="select-diamante">
+        <option value="" selected>Selecione uma opção</option>
+    </select>
+
+    <div class="mostra-diamante" id="mostra-diamante"></div>
```

Comment:
```suggestion
    <div class="mostra-diamante" id="mostra-diamante"></div>
```

Por que voce precisa de um ID e uma CLASSE com os mesmos nomes?
Não da pra usar um ou outro ?